packages/core/workers/src/WorkerFarm.js

```diff
@@ -516,6 +517,7 @@ export default class WorkerFarm extends EventEmitter {
 
     promises.push(this.localWorker[method](this.workerApi, ...args));
     await Promise.all(promises);
+    clearConfigCache();
```

Comment:
As mentioned, I think you can use this function to call the `clearConfigCache` function in each worker from parcel.js. Something like this:

```js
await this.#farm.callAllWorkers('clearConfigCache', []);
```

This will call the clearConfigCache function in each worker.